taxi_service/settings.py

```diff
@@ -143,3 +144,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK="bootstrap4"
```

Comment:
There is a duplicate setting for `CRISPY_TEMPLATE_PACK`. You should remove this line to avoid redundancy, as it is already defined on line 80.